lib/web/oidcidp.go

```diff
@@ -32,6 +32,8 @@ import (
 const (
 	// OIDCJWKWURI is the relative path where the OIDC IdP JWKS is located
 	OIDCJWKWURI = "/.well-known/jwks-oidc"
+	// OktaJWKSURI is the relative path where the Okta JWKS is located
```

Comment:
lol, is `OIDCJWKWURI` is a typo? (has a W instead of a S)

Reply:
looks like it, I just didn't want to increase the scope of this Okta work further by updating it